src/main/java/com/datapipe/jenkins/vault/credentials/common/VaultHelper.java

```diff
@@ -156,6 +164,13 @@ static String getVaultSecretKey(@NonNull String secretPath,
         }
     }
 
+    @CheckForNull
+    private static ItemGroup<Item> getItemGroup(String path) {
+        Item item = Jenkins.get().getItemByFullName(path);
+        return item instanceof ItemGroup ? (ItemGroup<Item>) item : null;
```

Comment:
@jglick is this really the best way to get an `ItemGroup`? This seems like Jenkins is lacking an way of properly grabbing an `ItemGroup` from a name.